frontend/src/components/Layout/Menu/styled.ts

```diff
@@ -1,4 +1,4 @@
-import styled from 'styled-components';
+import { styled } from '@mui/material';
 import { makeStyles } from '@material-ui/core';
```

Comment:
maybe it could be imported from mui instead? 
<img width="447" alt="image" src="https://user-images.githubusercontent.com/44285339/163812566-9e64ddc4-6bf0-4b11-b014-20b651ea2b9f.png">

Reply:
done